src/Checkout.php

```diff
@@ -85,7 +85,7 @@ public static function create($owner, array $sessionOptions = [], array $custome
         }
 
         // Make sure to collect address and name when Tax ID collection is enabled...
-        if (isset($data['customer']) && $data['tax_id_collection']['enabled'] ?? false) {
+        if (isset($data['customer']) && ($data['tax_id_collection']['enabled'] ?? false)) {
```

Comment:
Discovered that this condition previously didn't evaluate correctly without parenthesis, as the null coalescing operator `??` has a lower precedence than the `&&` operator, so the condition is equivalent to:

```php
(isset($data['customer']) && $data['tax_id_collection']['enabled']) ?? false
```